ui/css/logviewer.css

```diff
@@ -53,6 +53,7 @@ body {
 
 .text-danger {
     color: #db0000;
+    background: #fbe3e3 !important;
```

Comment:
Using !important is apparently bad: https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#The_!important_exception

Is there any way we could avoid using it?